cassovary-core/src/main/scala/com/twitter/cassovary/graph/ArrayBasedDirectedGraph.scala

```diff
@@ -225,7 +228,7 @@ object ArrayBasedDirectedGraph {
           if (nodeIdSet(id) == 1) {
             numNodes += 1
             if (table(id) == null) {
-              val node = ArrayBasedDirectedNode(id, ArrayBasedDirectedNode.noNodes, storedGraphDir)
+              val node = ArrayBasedDirectedNode(id, Array[Int](), storedGraphDir, neighborsSorted)
```

Comment:
I think Array[Int]() creates a new empty object everytime. That was the idea to have noNodes.